### ISSUE TITLE
Bug: ARC seal `instance` undocumented and results in `undefined` value, e.g. `i=undefined`

### ISSUE THREAD

**Issue by titanism** [postalsys/mailauth#75]
```txt
ARC-Authentication-Results: i=undefined; mx2.forwardemail.net;  // <------ YOU CAN SEE `i=undefined` here
 dkim=none (message not signed);
 spf=pass (mx2.forwardemail.net: domain of xxxxxx@xxxxxxx.com designates 1.2.3.4 as permitted sender)
 smtp.mailfrom=xxxxxx@xxxxxxx.com smtp.helo=xxxxxxx.com;
 dmarc=pass (p=NONE arc=none) header.from=xxxxxxx.com header.d=xxxxxxx.com;
 bimi=skipped (too lax DMARC policy)
```

It appears `i=undefined` is set always due to undocumented and missing `instance` option (?)

cc @andris9

**Comment by andris9** [comment]
It seems like a bug; it should default to 1 if there is no existing ARC header already set.

**Comment by andris9** [comment]
Fixed in v4.8.2.
